git_machete/cli.py

```diff
@@ -49,7 +49,7 @@
 commands_and_aliases = list(long_docs.keys()) + list(alias_by_command.keys())
 
 
-def get_help_description(command: str = None) -> str:
+def get_help_description(command: str = None, do_not_display_help_topics: bool = False) -> str:
```

Comment:
```suggestion
def get_help_description(command: str = None, display_help_topics: bool) -> str:
```
Drop the default value + do not use negations